region/maridia/outer/Mama Turtle Room.json

```diff
@@ -450,14 +510,34 @@
       ],
       "note": "Letting Mama Turtle sleep, climb the right wall using HiJump (with a down-grab) and escape the water with Space Jump ."
     },
+    {
+      "link": [1, 6],
```

Comment:
This is kind of the simplest/most straightforward [1, 6], so I moved it to be earlier in the order